code/game/objects/items/weapons/tools/stunbatons.dm

```diff
@@ -247,13 +247,13 @@
 
 /obj/item/tool/baton/slimebaton
 	name = "xenobio baton"
-	desc = "A stunbaton that is designed against slimes and other lab misstakes."
+	desc = "A stunbaton that is designed against slimes and other lab mistakes."
 	icon_state = "prod_si"
 	item_state = "prod_si"
 	force = WEAPON_FORCE_NORMAL
 	throwforce = WEAPON_FORCE_NORMAL
 	stunforce = 0
-	agonyforce = 50
+	agonyforce = 20 //No
```

Comment:
```suggestion
	agonyforce = 30
```
Its expensive, and you can do the same meme setups to 1 tap everyone (and better) with normal baton you can print with a sec disk/genetics free baton

The issue is that tool upgrades for hit cost directly also mult agonyforce as seen [here](https://github.com/sojourn-13/sojourn-station/blob/master/code/game/objects/items/weapons/tools/mods/_upgrades.dm#L371).